packages/shared/lib/models/Sync.ts

```diff
@@ -38,6 +38,7 @@ export interface Sync extends TimestampsAndDeleted {
         nanos?: number;
     };
     frequency: string | null;
+    last_fetched_at?: Date | null;
```

Comment:
It probably impact the creation type but a column can't be undefined.
```suggestion
    last_fetched_at: Date | null;
```

Reply:
Updated with https://github.com/NangoHQ/nango/pull/1956/commits/0d24d66585f7f380d1dbd9ba996aa84d5b4b464c. Yeah, forces us to be a bit more explicit